src/web/handlers.rs

```diff
@@ -53,7 +53,7 @@ pub async fn get_tiles(
     } else {
         Some(LocationResult::from_header(header, &state.settings))
     }
-    .unwrap_or_default();
+    .unwrap_or_else(|| LocationResult::from(&state.settings));
```

Comment:
nit: I think both the mmdb and `from_header` clauses always return `Some()`, so this line's superfluous if the clauses just return their unwrapped results

Reply:
Yeah, I converted the Result to a `?`